src/PendingCalls/UsesCall.php

```diff
@@ -84,8 +84,10 @@ public function in(string ...$targets): void
         }, $targets);
 
         $this->targets = array_reduce($targets, function (array $accumulator, string $target): array {
-            if (is_dir($target) || file_exists($target)) {
-                $accumulator[] = (string) realpath($target);
+            if (($matches = glob($target)) != false) {
```

Comment:
Perhaps use strict equality here?
```suggestion
            if (($matches = glob($target)) !== false) {
```